pretrain.py

```diff
@@ -14,31 +14,24 @@
 config = parser.parse_args()
 
 
-## Load the dataset ##
+## TODO: Load the dataset ##
 
 dataset = ...
 
 
-### Train the tokenizer
+### Train the tokenizer & Load the pretrained tokenizer
+# ! Multiple Possible Tokenizers
+# TODO: Find the best tokenizer
 
+bpe_tokenizer_trained = bpe_tokenizer()
```

Comment:
I would added an argument to argparse the chooses the tokenizer, and then load the correct one, rather than load 4 tokenizers.